### PR TITLE
Feature/misc banner updates

### PR DESCRIPTION
**Pull request by john-tco**
## Description

Ticket # and link

Misc updates related to GAP-2090

## Type of change

Please check the relevant options.

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] This change requires a documentation update.

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes:

- [ ] Unit Test

- [ ] Integration Test (if applicable)

- [ ] End to End Test (if applicable)

## Screenshots (if appropriate):

Please attach screenshots of the change if it is a UI change:

# Checklist:

- [ ] If I have listed dependencies above, I have ensured that they are present in the target branch.
- [ ] I have performed a self-review of my code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation where applicable.
- [ ] I have ran cypress tests and they all pass locally.

### CODE REVIEW

File: pages/notifications/manage-notifications/index.tsx [hunk 1]
```diff
@@ -334,6 +335,17 @@ const ManageNotifications = (props) => {
...
+  //The subscription-notification alert rendered by "urlAction" contains the same content as the successful migration banners.
+  //this variable hides the alert when one of the migrations is successful and neither of them have failed
+  const hideAlertConfirmationMessage =
+    shouldRenderMigrationBanner &&
+    props.migrationBannerProps.migrationType ===
+      LOGIN_NOTICE_TYPES.SUBSCRIPTION_NOTIFICATIONS &&
+    (props.migrationBannerProps.applyMigrationStatus === 'SUCCEEDED' ||
+      props.migrationBannerProps.findMigrationStatus === 'SUCCEEDED') &&
+    props.migrationBannerProps.applyMigrationStatus !== 'FAILED' &&
+    props.migrationBannerProps.findMigrationStatus !== 'FAILED';
```
Comment (dominicwest) [inline pages/notifications/manage-notifications/index.tsx:347]: Can we refactor this into a function with, I think they're called, guard clauses. I'm sure it works but its a nightmare understanding the logic here

Comment (john-tco) [reply pages/notifications/manage-notifications/index.tsx:347]: Do you mean optional chaining? the migrationBannerProps here are always going to be defined - we assign defaults above

Happy to extract to a fn

Comment (john-tco) [reply pages/notifications/manage-notifications/index.tsx]: cleaned that up a bit

File: pages/notifications/manage-notifications/index.tsx [hunk 2]
```diff
@@ -413,4 +419,29 @@ const ManageNotifications = (props) => {
...
+  shouldRenderMigrationBanner: boolean,
+) => {
+  const isSubscriptionNotificationMigration =
+    migrationBannerProps.migrationType ===
+    LOGIN_NOTICE_TYPES.SUBSCRIPTION_NOTIFICATIONS;
+
+  if (!shouldRenderMigrationBanner && isSubscriptionNotificationMigration)
+    return false;
+
+  const oneOfTheMigrationsPassed =
```
Comment (jgunnCO) [inline pages/notifications/manage-notifications/index.tsx:437]: nitpicking but this being `true` could also mean both migrations passed

```suggestion
  const migrationPassed =
```

Comment (john-tco) [reply pages/notifications/manage-notifications/index.tsx:437]: I almost went with `atLeastOneOfThe...`

Comment (john-tco) [reply pages/notifications/manage-notifications/index.tsx]: Have renamed that

**Review by dominicwest — APPROVED** [review]

**Pull request merged**